docs/protocol/register_packages_on-chain.md

```diff
@@ -0,0 +1,95 @@
+This guide describes how to register software packages in the Autonolas Protocol using the [protocol frontend](https://protocol.autonolas.network/). You can register three types of packages: **services**, **agents** and **components**.
+
+## How packages are registered
+
+To register a package in the protocol, it must have been already published into a remote (IPFS) registry. Read [how to publish packages](https://docs.autonolas.network/open-autonomy/guides/publish_fetch_packages/) with the {{open_autonomy}} framework.
```

Comment:
I would refer to a unit initially as well since we have it in our contracts code: "To register a package / unit in the protocol, ..."

Reply:
added a clarification.